src/patternfly/upgrade-examples/PopoverUpgradeExamples/examples/popover-upgrade-examples-example1.hbs

```diff
@@ -0,0 +1,16 @@
+{{#> popover-upgrade-examples}}
+<div class="popovers-pf">
+  <div class="container-fluid container-popovers-pf">
+   <div class="popover fade right in" role="tooltip" style="display: block; position:relative;">
+      <div class="arrow" style="top: 50%;"></div>
+        <h3 class="popover-title closable">Popover Header
+          <button type="button" class="close" aria-hidden="true"><span class="pficon pficon-close"></span></button>
+        </h3>
+      <div class="popover-content"><b>Popover Body</b> <br> <br> This is more information about a smartproxy. This is more information about a smartproxy. This is more information about a smartproxy. This is more information about a smartproxy.</div>
```

Comment:
Can you change the body of the examples for both to be: 
"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec fringilla turpis." like in the pf4 component.

Reply:
Lorem ipsum content has been added.